che-mount/sync.sh

```diff
@@ -60,20 +74,35 @@ error_exit() {
 }
 
 # See: https://sipb.mit.edu/doc/safe-shell/
-set -e
 set -u
 
 init_logging
 init_global_variables
 parse_command_line "$@"
 
+info "INFO: ECLIPSE CHE: Mounting user@$1:/projects with SSHFS"
 sshfs user@$1:/projects /mntssh -p $2
-unison /mntssh /mnthost -batch -fat -silent -auto -prefer=newer > /dev/null 2>&1
-
+status=$?
+if [ $status -ne 0 ]; then
+    error "ERROR: Fatal error occurred ($status)"
+    exit 1
+fi
 info "INFO: ECLIPSE CHE: Successfully mounted user@$1:/projects"
- 
-while :
-do
-    unison /mntssh /mnthost -batch -fat -silent -auto -prefer=newer > /dev/null 2>&1
-    sleep 1
-done
+info "INFO: ECLIPSE CHE: First syncronisation is happening now and can take a long time. Please wait."
+unison /mntssh /mnthost -batch -fat -silent -auto -prefer=newer -log=false > /dev/null 2>&1
+status=$?
+if [ $status -ne 0 ]; then
+    error "ERROR: Fatal error occurred ($status)"
+    exit 1
+fi
+info "INFO: ECLIPSE CHE: First syncronisation completed."
+info "INFO: ECLIPSE CHE: Now synchronization happens continuously every ${UNISON_REPEAT_DELAY_IN_SEC} seconds."
```

Comment:
Change to "INFO: ECLIPSE CHE: Background sync continues every ${UNISON_REPEAT_DELAY_IN_SEC} seconds."